images/linux/Ubuntu1604-README.md

```diff
@@ -179,6 +179,7 @@ The following software is installed on machines in the Hosted Ubuntu 1604 pool
 - Android NDK 19.0.5232133
 - Android ConstraintLayout 1.0.2
 - Android ConstraintLayour 1.0.1
+- Vcpkg 2018.11.23-nohash
```

Comment:
Looks like there is missing empty line

Reply:
Also, this should not be part of resultant PR (docs are updated as part of image generation process)